scripts/applyPatches.sh

```diff
@@ -73,37 +81,26 @@ function applyPatch {
     fi
 }
 
-# Move into spigot dir
-cd "$workdir/Spigot"
-basedir=$(pwd)
 # Apply Spigot
 (
-    applyPatch ../Bukkit Spigot-API HEAD &&
+    cd "$workdir/Spigot"
+    basedir="$(pwd)"
+    applyPatch ../Bukkit Spigot-API HEAD
     applyPatch ../CraftBukkit Spigot-Server patched
-) || (
-    echo "Failed to apply Spigot Patches"
-    exit 1
-) || exit 1
-# Move out of Spigot
-basedir="$1"
-cd "$basedir"
+) || (echo "Failed to apply Spigot Patches"; return 1)
```

Comment:
(minor) Don't see the reason for a subshell for these commands.
```suggestion
) || {echo "Failed to apply Spigot Patches"; return 1;}
```